src/windows/main_window.py

```diff
@@ -1256,6 +1276,20 @@ def actionPreviousMarker_trigger(self, event):
                 all_marker_positions.append(selected_tran.data["position"])
                 all_marker_positions.append(selected_tran.data["position"] + (selected_tran.data["end"] - selected_tran.data["start"]))
 
+        # remove duplicates
+        all_marker_positions = list(set(all_marker_positions))
```

Comment:
I realized — one thing we should watch for in testing in case it's an issue, though I don't think it's anything that needs to be addressed until that time: We could end up with some duplicates remaining on the list, because these are floating-point values so "equal" points won't always come out exactly equal.

In practice I don't think it'll matter 99% of the time, since even if there are multiples they should all translate back to the same single frame number for seeking purposes. Though I suppose if the user's current position is bordered by other not-quite-equal positions, worst case they might have to skip an extra time or two to get off that position and jump to the actual next point.

Reply:
I haven't investigated why, but in practice even without this duplicate removal I found no case of having to iterate several times on the same point, even with multiple elements on one point. I'm guessing because we navigate one right or one left searching for nearest non equal. But as you say it could possibly happen also with floating points.

Reply:
Oh, yeah, the duplicate removal isn't meant to affect seeking — you'll always be looking for the _next_ lower/higher time value to your current position. So it wouldn't matter if there are 1 or 27 duplicates of that next position — or even your current position — on the list. There'll still be a next adjacent position.

The duplicate removal is just to make the list shorter, because otherwise you'd end up sorting/searching a list filled with literally two dozen `0`s for **each** selected clip. :laughing: 

If the floating-point thing does prove to be an issue, duplicate removal isn't going to solve it anyway, since the whole thing there is that "equal" values may _not_ be **equal**-equal. But I'm really not worried about that, unless it actually comes up. For now I'm content to file it away as something to watch out for, nothing more.

Reply:
(Where "`0`" is the clip's `data["position"]` on the timeline, not necessarily `0`.)